protocols/gossipsub/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## 0.45.0 
+## 0.45.1 - unreleased
```

Comment:
recently (today) I learned you need to change the `Cargo.toml` upgrading the version